Assignment-10-August-22/userprofile/src/Components/ButtonComponent.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Button } from "@mui/material";
+
+const ButtonComponent = ({
+  label,
+  onClick,
+  color = "primary",
+  variant = "contained",
+  ...props
+}) => {
```

Comment:
if we are simply returning the component then we can also use shorthand operator

```suggestion
}) => (
    <Button onClick={onClick} color={color} variant={variant} {...props}>
      {label}
    </Button>
  );
```